qa/tasks/cephfs/test_volumes.py

```diff
@@ -162,6 +162,15 @@ def test_subvolume_create_with_auto_cleanup_on_fail(self):
         else:
             raise
 
+    def test_subvolume_create_with_invalid_size(self):
+        # create subvolume with an invalid size -1
+        subvolume = self._generate_random_subvolume_name()
+        try:
+            self._fs_cmd("subvolume", "create", self.volname, subvolume, "--size", -1)
+        except CommandFailedError as ce:
+            if ce.exitstatus != errno.ENOENT:
```

Comment:
s/errno.ENOENT/errno.EINVAL/ ?

Reply:
Then it won't raise here I think, as we are raising EINVAL in  [subvolume.py](https://github.com/ceph/ceph/pull/30058/files#diff-9455d141cc54cc29b319d9e6a29678c8R83)

Reply:
The exit status should be EINVAL, right? ENOENT doesn't make sense here. Maybe you've to change the code in subvolume.py,
```
raise VolumeException(-e.args[0], "Invalid size: '{0}'".format(size))
```
 to
```
raise VolumeException(-errno.EINVAL, "Invalid size: '{0}'".format(size))
```